api/src/services/assets.ts

```diff
@@ -86,16 +105,28 @@ export class AssetsService {
 		 */
 		if (!isValidUuid(id)) throw new ForbiddenError();
 
-		if (systemPublicKeys.includes(id) === false && this.accountability) {
-			await validateAccess(
+		let allowedFields: string[] | undefined;
+
+		if (!systemPublicKeys.includes(id) && this.accountability && !this.accountability.admin) {
```

Comment:
Let's simplify this.
```suggestion
		if (!systemPublicKeys.includes(id) && this.accountability?.admin !== true) {
```

Reply:
Okay, but then I need to cast to a non-null assertion. But yes, we can do it that way.